data-model/src/main/scala/za/co/absa/enceladus/model/properties/PropertyDefinitionStats.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.model.properties
+
+import za.co.absa.enceladus.model.properties.essentiality.Essentiality
+import za.co.absa.enceladus.model.properties.propertyType.PropertyType
+
+case class PropertyDefinitionStats(name: String,
+                                   version: Int = 1,
+                                   propertyType: PropertyType,
+                                   essentiality: Essentiality = Essentiality.Optional,
+                                   disabled: Boolean = false,
+                                   missingInDatasetsCount: Int = 0
+) {
+  def setName(value: String): PropertyDefinitionStats = this.copy(name = value)
+  def setVersion(value: Int): PropertyDefinitionStats = this.copy(version = value)
+  def setPropertyType(value: PropertyType): PropertyDefinitionStats = this.copy(propertyType = value)
```

Comment:
Do I need property type in a full or just name?

Reply:
Now that I started on the UI I realized that on the side panel we do not need to display many things, just the name, essenttiality and in how many datasets is missing, then when clicking on one of those there would be another request to /api/properties/datasets/{propertyName} (in PropertyDefinitionController.scala) in order to get the data needed in the info tab. Does it make sense?